src/service/chat.js

```diff
@@ -132,11 +132,11 @@ const sendAnonymousDMOtherProfile = async ({
 
 const sendSignedDMOtherProfile = async ({user_id, message}) => {
   const payload = {
-    members: [user_id],
+    member: user_id,
     message
   };
 
-  const response = await api.post('/chat/init-chat', payload);
+  const response = await api.post('/chat/init-chat-signed-v2', payload);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The endpoint for initializing signed chat has been changed from '/chat/init-chat' to '/chat/init-chat-signed-v2'. Ensure that the new endpoint is correctly configured and tested on the server side.
```diff
-  const response = await api.post('/chat/init-chat', payload);
+  const response = await api.post('/chat/init-chat-signed-v2', payload);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->